deluge/ui/gtkui/filtertreeview.py

```diff
@@ -244,20 +241,24 @@ def get_pixmap(self, cat, value):
 
         if pix:
             try:
-                return gtk.gdk.pixbuf_new_from_file(get_pixmap("%s16.png" % pix))
+                return GdkPixbuf.Pixbuf.new_from_file(get_pixmap("%s16.png" % pix))
             except GError as ex:
                 log.warning(ex)
         return self.get_transparent_pix(16, 16)
 
     def get_transparent_pix(self, width, height):
-        pix = gtk.gdk.Pixbuf(gtk.gdk.COLORSPACE_RGB, True, 8, width, height)
-        pix.fill(0x0000000)
-        return pix
+        # pix = GdkPixbuf.Pixbuf(GdkPixbuf.Colorspace.RGB, True, 8, width, height)
+        # pix.fill(0x0000000)
+        # return pix
+
+        # Should really give back a properly size pixbuf...
```

Comment:
What does all this mean, is it fixed or not? Is the `icon_empty` code required or just a nice addition?